packages/__examples__/.storybook/preview.js

```diff
@@ -48,54 +35,9 @@ addParameters({
       name: 'instructure-ui',
       brandUrl: 'https://instructure.design',
       brandImage:
-        'https://instructure.design/icons-bea652a7d0546b99c7cba9d3b4a538ee/favicon-32x32.png'
+        'https://instructure.design/icons-bea652a7d0546b99c7cba9d3b4a538ee/favicon-32x32.png' // TODO this URL is broken
     }),
     showPanel: false,
     isFullscreen: false
   }
 })
```

Comment:
again, the new format, see https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#using-previewjs